Editor/Window/ConnectToFleetInput.cs

```diff
@@ -0,0 +1,205 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Amazon.GameLift.Model;
+using AmazonGameLift.Editor;
+using Editor.CoreAPI;
+using UnityEngine.UIElements;
+
+namespace Editor.Window
+{
+    public class ConnectToFleetInput : StatefulInput
+    {
+        private static IReadOnlyCollection<VisualElement> _fleetVisualElements;
+        
+        public string FleetId;
+        
+        private static readonly List<string> s_fleetNameList = new();
+        
+        private TextField _fleetNameInput;
+        private DropdownField _fleetNameDropdownContainer;
+        private VisualElement _fleetCreateFoldout;
+        private VisualElement _fleetConnectFoldout;
+        private VisualElement _fleetId;
+        private Label _fleetIdText;
+        private VisualElement _fleetStatus;
+        private readonly GameLiftFleetManager _fleetManager;
+        private readonly StateManager _stateManager;
+        private Button _cancelButton;
+        
+        private FleetStatus _fleetState;
+        private List<FleetAttributes> _fleetsList;
+
+        public ConnectToFleetInput(VisualElement container, StateManager stateManager, FleetStatus initialState)
```

Comment:
I believe Jackson pointed out in one of the other PRs #101, that these need to be factored out in their own uxml, and not be inline in the AnywherePage.uxml. The Anywhere page class should instantiate the ConnectToFleetInput class which then loads up the related uxml. We should follow the same here.

Reply:
Separated out to different uxml